app/javascript/components/mentoring/automation/useAutomation.tsx

```diff
@@ -65,11 +66,27 @@ export function useAutomation(
       BLANK_TRACK_DATA
   )
 
-  const { status, error, resolvedData, latestData, isFetching } =
-    usePaginatedRequestQuery<APIResponse>(
-      ['mentor-representations-list', request],
-      request
-    )
+  const CACHE_KEY = [
+    'mentor-representations-list',
+    ...Object.values(request.query),
+  ]
+
+  const queryClient = useQueryClient()
+
+  queryClient.setQueryData([CACHE_KEY], request.options.initialData)
```

Comment:
Why explicitly set the initial data here? I think we use that lots in our code, but this seems to be the only place you explicitly set it.